lib/ansible/modules/storage/netapp/na_ontap_volume.py

```diff
@@ -86,6 +85,7 @@
   junction_path:
     description:
     - Junction path of the volume.
+    - To unmount, use junction path ''.
```

Comment:
```suggestion
    - To unmount, use junction path C('').
```

https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation